Delete User methods

davidor opened the pull request:
End-users are no longer supported in Apisonator: https://github.com/3scale/apisonator/pull/128

davidor committed:
Delete User

No longer supported in Apisonator

unleashed commented:
This would render newer versions unable to work with older versions of 3scale. Can we keep this for a while or are all supported versions already not having these?

unleashed reviewed (commented):
Ok whenever we decide this can be merged.

davidor committed:
service: stop raising ServiceRequiresDefaultUserPlan

This cannot longer be raised. Users are no longer supported, so there
are no user plans.

davidor commented:
The feature of end-users was deprecated many versions ago, and it disappeared from the codebase some months ago, meaning that calling any of those methods in a recent version of Apisonator results in a 404 error. I'd say we should go ahead and document it in the changelog, although in practice, this is not very important, because the only user of this library is Porta and they stopped using this feature months ago too.
